React/my-app/src/service/accounts.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://wbooks-api-stage.herokuapp.com/api/v1/",
```

Comment:
considerate that baseURL and all sensitive data in a proyect always is spoted into a `.env` or environment file.
This file is local and never should be uploaded for security reasons.